modules/mvt/src/mvt-loader.js

```diff
@@ -12,8 +12,7 @@ export const MVTWorkerLoader = {
   category: 'geometry',
   options: {
     mvt: {
-      geojson: true,
-      tileProperties: {},
```

Comment:
Nit: Why no default for `_tileProperties`?

Reply:
I thought that as there was no `geojson` property, `tileProperties` were useless, but I think it is good to keep that default value as well.